### PR TITLE
POR node operators documentation

### PR DESCRIPTION
**Pull request by bayram98**
This PR is for add node operators documentation for POR service.

### CODE REVIEW

File: en/SUMMARY.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 * [Orakl Network Fetcher](node-operators-guide/fetcher.md)
 * [Orakl Network Data Feed](node-operators-guide/data-feed.md)
 * [Orakl Network Delegator](node-operators-guide/delegator.md)
+* [Orakl Network PoR](node-operators-guide/por.md)
```
Comment (martinkersner) [inline en/SUMMARY.md:30]: Let's keep it in the long form so people can have better chance to understand.

```suggestion
* [Orakl Network Proof of Reserve](node-operators-guide/por.md)
```

File: en/node-operators-guide/por.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Every proof of reserve has configuration that describes the least frequent update interval called `heartbeat`, and minimum deviation threshold (`deviationTreshold`). 
+
+The **Orakl Network Proof of Reserve** operates as a streamlined, single-process system, easily triggered by a Cron job. The initial phase involves retrieving the most recent `roundId` and `POR value` from the `POR contract`. Subsequently, the process assesses eligibility for submission through a combination of the `Heartbeat Check` and `Deviation Check`. If either condition is met, the final step involves fetching data from the `API resource` and reporting the next round to the `POR contract`. This cohesive workflow ensures efficient and timely execution of the `Proof of Reserve` process.
+
+
+The code is located under [`core` directory](https://github.com/Bisonai/orakl/tree/master/core).
```
Comment (martinkersner) [inline en/node-operators-guide/por.md:15]: Let's link directly to PoR directory.

File: en/node-operators-guide/por.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    --file-path ${adapterJsonFile}
+
+orakl-cli aggregator insert \
+    --chain ${chain} \
+    --file-path ${aggregatorJsonFile}
+```
+
+## Configuration
+
+Before we launch the **Orakl Network POR**, we must specify [several environment variables](https://github.com/Bisonai/orakl/blob/master/core/.env.example). The environment variables are automatically loaded from a `.env` file.
```
Comment (martinkersner) [inline en/node-operators-guide/por.md:47]: Let's use full name.

```suggestion
Before we launch the **Orakl Network Proof Of Reserve**, we must specify [several environment variables](https://github.com/Bisonai/orakl/blob/master/core/.env.example). The environment variables are automatically loaded from a `.env` file.
```

File: en/node-operators-guide/por.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+* `POR_AGGREGATOR_HASH`
+* `NODE_ENV=production`
+* `CHAIN`
+* `PROVIDER_URL`
+* `ORAKL_NETWORK_API_URL`
+* `LOG_LEVEL`
+* `HEALTH_CHECK_PORT`
+* `SLACK_WEBHOOK_URL`
+
+`POR_AGGREGATOR_HASH` first environment variable is to define Aggregator Hash value of the unique aggregator which will used for Proof of Reserve service.
```
Comment (martinkersner) [inline en/node-operators-guide/por.md:58]:
```suggestion
`POR_AGGREGATOR_HASH` first environment variable is to define aggregator hash value of the unique aggregator which will used for Proof of Reserve service.  
```

File: en/node-operators-guide/por.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+# Orakl Network Proof of Reserve
+
+## Description
+
+The **Orakl Network Proof of Reserve** is a critical component within the Orakl Network ecosystem, designed to establish and verify the reserve holdings of financial entities. This service ensures transparency and trust in the financial operations by providing a secure and auditable process for validating reserve data. The Orakl Network Proof of Reserve is a cornerstone within the ecosystem, dedicated to establishing trust in financial operations. This service seamlessly integrates off-chain and on-chain processes, providing a robust framework to verify and authenticate reserve holdings.
+
+
+A Proof of Reserve is defined by a pair of an adapter and an aggregator, and can be accessed on-chain through a `POR Proxy` smart contract. The `POR Proxy` is an auxiliary contract that redirects read requests to `POR` contract. The `POR` contract holds all submission values that is served to consumers through `POR Proxy` contract.
```
Comment (martinkersner) [inline en/node-operators-guide/por.md:8]: Let's say that the on-chain implementation is same as for Data Feed. And the we will reference to real names of smart contracts `Aggregator` and `AggregatorProxy`. 🙌

**Review by martinkersner — APPROVED** [review]
LGTM! I just have couple recommendation in comments.

File: en/node-operators-guide/por.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+The **Orakl Network Proof of Reserve** operates as a streamlined, single-process system, easily triggered by a Cron job. The initial phase involves retrieving the most recent `roundId` and `POR value` from the `POR contract`. Subsequently, the process assesses eligibility for submission through a combination of the `Heartbeat Check` and `Deviation Check`. If either condition is met, the final step involves fetching data from the `API resource` and reporting the next round to the `POR contract`. This cohesive workflow ensures efficient and timely execution of the `Proof of Reserve` process.
+
+
+The code is located under [`core` directory](https://github.com/Bisonai/orakl/tree/master/core).
+
+## Setup
+
+The **Orakl Network Proof of Reserve** requires an access to configuration of adapter, aggregator and reporter.
+
```
Comment (nick-bisonai) [inline en/node-operators-guide/por.md:20]: maybe it should start with adding service `POR`?
so
```
orakl-cli service insert --name POR
```

**Pull request merged**